FlowCrypt/Controllers/Setup/SetupEKMKeyViewController.swift

```diff
@@ -98,11 +98,14 @@ extension SetupEKMKeyViewController {
                 source: .ekm,
                 for: self.user.email
             )
-            allFingerprints.append(contentsOf: parsedKey.keyDetails.flatMap { $0.fingerprints })
+            allFingerprints.append(contentsOf: parsedKey.keyDetails.map(\.fingerprints))
         }
         if self.storageMethod == .memory {
-            let passPhrase = PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: allFingerprints.unique())
-            try appContext.passPhraseService.savePassPhrase(with: passPhrase, storageMethod: self.storageMethod)
+            try allFingerprints
+                    .map { PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: $0) }
+                    .forEach {
+                        try appContext.passPhraseService.savePassPhrase(with: $0, storageMethod: storageMethod)
+                    }
         }
```

Comment:
I'd like to ask that we standardize on a `for thing in things {` loop and always avoid `.forEach {`. It's more direct: you can return from it, you can try/throw/catch without having to try in front of the loop, and why it "looks less functional" I'd say it's more idiomatic. Either way, it's one of those small things I'll have to insist on 🤷 

So in this case, 


```swift
for allFingerprintsOfOneKey in allFingerprintsOfAllKeys {
    try appContext.passPhraseService.savePassPhrase(
        with: PassPhrase(value: passPhrase, fingerprintsOfAssociatedKey: allFingerprintsOfOneKey) },
        storageMethod: storageMethod
    )
}
```

It will be clearer.